### DISCUSSION TITLE
WARNING Failed to search registry with scope "mmyolo" in the "log_processor" registry tree

### DISCUSSION THREAD

**Discussion by vansin** [open-mmlab/mmyolo#122] [Q&A]
10/01 09:23:01 - mmengine - WARNING - Failed to search registry with scope "mmyolo" in the "log_processor" registry tree. As a workaround, the current "log_processor" registry in "mmengine" is used to build instance. This may cause unexpected failure when running the built modules. Please check whether "mmyolo" is a correct scope, or whether the registry is initialized.

**Comment by amxl56** [comment]
How was it solved?
